src/Components/Steps/Disclaimer/Disclaimer.tsx

```diff
@@ -0,0 +1,218 @@
+              />
+              <a
+                className="link-color"
+                href={publicChargeOption.link}
+                onClick={() => {
+                  dataLayerPush({
+                    event: 'public_charge',
+                    action: 'public charge link click',
+                  });
+                }}
+              >
+                <FormattedMessage
+                  id="landingPage.publicChargeLink"
+                  defaultMessage="Colorado Department of Human Services Public Charge Rule"
+                />
+              </a>
+              <FormattedMessage id="landingPage.publicCharge.afterLink" defaultMessage="." />
+            </div>
+          </li>
+        </ul>
+      </CardContent>
+    );
+  };
+
+  const getLinksForCheckbox = () => {
+    if (locale in privacyLink && locale in consentToContactLink) {
+      return {
+        privacyPolicyLink: privacyLink[locale],
+        addTermsConsentToContact: consentToContactLink[locale],
+      };
+    } else {
+      return {
+        privacyPolicyLink: privacyLink['en-us'],
+        addTermsConsentToContact: consentToContactLink['en-us'],
+      };
+    }
+  };
+
+  const createAgreeTTSCheckboxLabel = () => {
+    return (
+      <div className="landing-pg-font">
+        <FormattedMessage
+          id="disclaimer-label"
+          defaultMessage="By proceeding, you confirm that you have read and agree to the "
+        />
+        <a href={getLinksForCheckbox().privacyPolicyLink} target="_blank" className="link-color">
+          <FormattedMessage id="landingPage-policyText" defaultMessage="Privacy Policy" />
+        </a>
+        <FormattedMessage id="landingPage-and-text" defaultMessage=" and " />
+        <a href={getLinksForCheckbox().addTermsConsentToContact} target="_blank" className="link-color">
+          <FormattedMessage id="landingPage-additionalTerms" defaultMessage="Additional Terms & Consent to Contact" />
+        </a>
+        <FormattedMessage id="landingPage-disclaimer-lable-end" defaultMessage="." />
+      </div>
+    );
+  };
+
+  const renderCheckboxHelperText = () => {
+    return (
+      <ErrorMessageWrapper fontSize="1rem">
+        <FormattedMessage id="disclaimer.error" defaultMessage="Please check the box to continue." />
+      </ErrorMessageWrapper>
+    );
+  };
+
+  return (
+    <main className="benefits-form">
+      <QuestionHeader>
+        <FormattedMessage id="disclaimer.header" defaultMessage="What you should know: " />
+      </QuestionHeader>
+      {renderDisclaimerText()}
+      <form onSubmit={handleSubmit(formSubmitHandler)}>
+        <div className="checkbox-container">
+          <Controller
+            name="agreeToTermsOfService"
+            control={control}
+            rules={{ required: true }}
```

Comment:
Will zod enforce this?

Reply:
Yes.

Reply:
Just wondering what happens if you remove this line. Will the form still submit, or will zod catch that there is invalid data?